### PR TITLE
Complete csvs

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR add some missing fields into datasets.csv.

It also provide a reusable helper to fetch nested attributes/keys on any object

### CODE REVIEW

File: udata/utils.py [hunk 1]
```diff
@@ -175,3 +175,25 @@ def to_bool(value):
...
+    Get an attribute or a key recursively.
+
+    :param obj: The object to fetch attribute or key on
+    :type obj: object|dict
+    :param key: Either a string in dotted-notation ar an array of string
+    :type key: string|list|tuple
+    '''
+    if not obj or not key:
+        return
+    parts = key.split('.') if isinstance(key, basestring) else key
```
Comment (davidbgk) [inline udata/utils.py]: `key.split('.', 1)` is probably more interesting here.

Comment (davidbgk) [inline udata/utils.py]: Well, not that much actually.

File: udata/utils.py [hunk 2]
```diff
@@ -175,3 +175,25 @@ def to_bool(value):
...
+    key = parts[0]
```
Comment (davidbgk) [inline udata/utils.py]: Using `parts.pop(0)` makes it possible to ease the end of the test.

**Comment by davidbgk** [comment]
r+wc

File: udata/utils.py (hunk 1 above)
Comment (noirbizarre) [inline udata/utils.py]: Yep, I'm splitting only once

File: udata/utils.py (hunk 2 above)
Comment (noirbizarre) [inline udata/utils.py]: OK, I will try that

**Pull request merged**